libevmasm/SimplificationRules.cpp

```diff
@@ -141,7 +141,8 @@ string Pattern::toString() const
 	switch (m_type)
 	{
 	case Operation:
-		s << instructionInfo(m_instruction).name;
+		//TODO:prevrandao: output difficulty or prevrandao? Make evmVersion member of Pattern?
```

Comment:
This is "unused" anyways, resp. it's only used if you change a hardcoded ``if (false)`` in ``ExpressionClasses.cpp`` for debugging output - so default evm version is perfectly fine.